src/gui/mzroll/mzroll.pro

```diff
@@ -45,6 +45,14 @@ win32 {
     LIBS += -lgoogle-breakpad -pthread
 }
 
+mac {
+    INCLUDEPATH  += $$top_srcdir/3rdparty/google-breakpad/src/
+    QMAKE_LFLAGS += -L$$top_builddir/libs/
+    LIBS += /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
```

Comment:
@rish9511 Not sure what this is. Can you add comments in the code?

Reply:
@sahil21  google-breakpad uses native apis. hence we need to link against those on all the platforms